src/mixins/KonvaRegion.js

```diff
@@ -103,6 +105,16 @@ export const KonvaRegionMixin = types.model({})
 
         if (e) e.cancelBubble = true;
 
+        if (isFF(FF_DBLCLICK_DELAY)) {
+          const isInDoubleClickWindow = lastClick && (ev.timeStamp - lastClick.time < DOUBLE_CLICK_MS_WINDOW);
+          const isSamePosClick = lastClick && (Math.abs(ev.offsetX - lastClick.x) < MAX_CLICK_POS_DEVIATION && Math.abs(ev.offsetY - lastClick.y) < MAX_CLICK_POS_DEVIATION);
+
+          if (isInDoubleClickWindow && isSamePosClick) {
```

Comment:
this is a cool approach but involves checking time stamps to confirm if double click happened or not - is there a reason we didnt use `ev.detail`? if `ev.detail = 1` then its 1 click, `ev.detail = 2` means double click

Reply:
Wow. Somehow I literally didn't know about that detail property. Thanks a lot! It makes my life easier.